lib/opFns.js

```diff
@@ -288,6 +288,31 @@ module.exports = {
       cb(null)
     })
   },
+  EXTCODEHASH: function (address, runState, cb) {
+    if (!runState._common.gteHardfork('constantinople')) {
+      trap(ERROR.INVALID_OPCODE)
+    }
+    var stateManager = runState.stateManager
+    address = addressToBuffer(address)
+
+    stateManager.getAccount(address, function (err, account) {
+      if (err) return cb(err)
+
+      console.log(account)
```

Comment:
Should be removed before merging :)

Reply:
ahhhh, will do

Reply:
With the new linting rules worked out in the RLP TypeScript transition prototype implementation https://github.com/ethereumjs/rlp there will be a rule preventing this (left ``console.log`` messages in release) altogether.

Actually there will be a lot of changes (hopefully improvements) to the toolchain once we role this out more broadly, you guys are very much invited to also have a look and join the discussion if you have some capacity, think the more eyes are on this the better. Will also make some mentioning of this in the next monthly recap blog post (probably today).